publish/publish.proj

```diff
@@ -262,39 +262,32 @@
   </Target>
 
   <Target Name="DoPushCoreHostPackagesToAzure"
-          DependsOnTargets="PublishToAzureBlobFeed;PublishSymbolsToAzureBlobFeed"
+          DependsOnTargets="PublishToAzureBlobFeed"
           Condition="'$(OfficialPublish)' == 'true'"/>
 
   <Target Name="PublishToAzureBlobFeed">
     <Error Condition="'$(PackagesUrl)'==''" Text="Missing property PackagesUrl" />
     <Error Condition="'$(TransportFeedAccessToken)'==''" Text="Missing property TransportFeedAccessToken" />
     <PropertyGroup>
       <ExpectedFeedUrl>$(PackagesUrl)</ExpectedFeedUrl>
+      <AssetManifestFilePath>$(BUILD_SOURCESDIRECTORY)/artifacts/log/AssetManifest/Manifest.xml</AssetManifestFilePath>
     </PropertyGroup>
     <ItemGroup>
       <ItemsToPush Remove="@(ItemsToPush)" />
       <ItemsToPush Include="@(_DownloadedStandardPackages)"/>
+       <ItemsToPush Include="@(_DownloadedSymbolsPackages)"/>
```

Comment:
Symbols and Standard nuget packages are published to same feed, thus, simplifying here.